lib/membrane_http_adaptive_stream/sink.ex

```diff
@@ -83,15 +83,15 @@ defmodule Membrane.HTTPAdaptiveStream.Sink do
         """
       ],
       segment_duration: [
-        spec: Manifest.SegmentDuration.t(),
+        spec: Membrane.Time.t(),
         description: """
-        The expected minimum and target duration of media segments produced by this particular track.
+        The expected target duration of media segments produced by this particular track.
```

Comment:
I would also point out in the description or by changing the option name that the segment won't be shorter than 'segment_description'.

Reply:
Judging by the [CMAF docs](https://hexdocs.pm/membrane_mp4_plugin/0.21.0/Membrane.MP4.Muxer.CMAF.html#module-chunk-creation), I believe that the duration of a segment can be shorter that this parameter.